### DISCUSSION TITLE
No longer ask users for scratch user id

### DISCUSSION THREAD

**Discussion by Shluffy** [theforumhelpers/theforumhelpers.github.io#126] [Ideas]
Once we have the username, we can use that to get the user's id from the scratch api.

**Comment by Accio1** [comment]
Requests to Scratch's API are fairly tedious because they require a CORS proxy. Plus, to load the page, we would have to make 100+ requests to the Scratch API within a few seconds, and the Scratch Team doesn't like multiple requests in a short time frame. Therefore, I'm not sure how realistic this idea is.

**Reply by Shluffy** [reply]
okay

**Comment by FunctionalMetatable** [comment]
fluffyscratch has a profile picture by username api.

**Comment by Steve0Greatness** [comment]
Why do we need the ID again?

**Reply by Shluffy** [reply]
to get the pfp of the user

**Reply by Steve0Greatness** [reply]
> to get the pfp of the user

ok...
